_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -52,6 +52,7 @@ class HealthCareApp extends React.Component {
     this.context.store.dispatch(ensureFieldsInitialized(path));
     if (validations.isValidSection(path, sectionData)) {
       hashHistory.push(this.getUrl('next'));
+      sectionData.sectionComplete = true;
```

Comment:
Reading the (docs)[http://redux.js.org/docs/basics/Actions.html] again, I think we do need to dispatch an action here: "Actions are payloads of information that send data from your application to your store. They are the only source of information for the store. You send them to the store using store.dispatch()."